classes/ObjectModel.php

```diff
@@ -232,6 +232,11 @@ public function __construct($id = null, $id_lang = null, $id_shop = null)
             $entity_mapper = Adapter_ServiceLocator::get("Adapter_EntityMapper");
             $entity_mapper->load($id, $id_lang, $this, $this->def, $this->id_shop, self::$cache_objects);
         }
+	
+	// @hook actionObject*Init
+	Hook::exec('actionObjectInit', array('object' => $this));
```

Comment:
Hi,

Thank you for you Pr. I believe you have a different indentation, are you using spaces or tabs ?

Thank you. Regards